pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -230,13 +231,115 @@ func (adc *attachDetachController) Run(stopCh <-chan struct{}) {
 	// 	return
 	// }
 
+	err := adc.populateActualStateOfWorld()
+	if err != nil {
+		glog.Errorf("Error populating the actual state of world: %v", err)
```

Comment:
All the rest of the attachdetach controller uses `glog.Errorf`. That's why I'm using it too. I'd rather stay consistent with the existing code and then eventually refactor all the error cases (including the old ones). I'd prefer to do that in a separate PR.